proposed/extended-coding-style-guide.md

```diff
@@ -29,21 +29,30 @@ open to interpretation. This PSR therefore seeks to clarify the content of PSR-2
 a more modern context with new functionality available, and make the errata to PSR-2
 binding.
 
+### Overview
+
+Throughout this document, any instructions MAY be ignored if they do not exist in versions
+of PHP supported by your project.
+
 ### Example
 
 This example encompasses some of the rules below as a quick overview:
 
 ```php
 <?php
+declare(strict_types=1);
```

Comment:
I think it would be more likely to catch people out to be honest as people will consider this as a string they need to include, rather than thinking about what it actually means.